### PR TITLE
[DO-1293] Implemented quality alerts on dashboard

### PR DESCRIPTION
**Pull request by viannbreslerAccso**
Implemented the quality alerts section on dashboard. This also affects the work for https://cofinity-x.atlassian.net/browse/DO-1292

### CODE REVIEW

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html [hunk 1]
```diff
@@ -19,74 +19,60 @@
...
+
+  <section class="dashboard--section">
     <mat-card>
       <mat-card-header>
         <div class="inline-block w-full">
           <div class="flex justify-between items-center">
-            <h3 class="mb-4 flex items-center">
+            <h3 class="pl-2 mb-4 flex items-center">
               <img class="pr-2" src="/assets/images/icons/qualityInvestigations.svg" />
               {{ 'pageTitle.investigations' | i18n }}
```
Comment (JoelDietz) [inline frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:74]: Color of the title is #111111 but should be #262124

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html [hunk 2]
```diff
@@ -98,27 +84,56 @@ <h3 class="mb-4 flex items-center">
...
+  </section>
+
+  <section class="dashboard--section">
+    <mat-card>
+      <mat-card-header>
+        <div class="inline-block w-full">
+          <div class="flex justify-between items-center">
+            <h3 class="pl-2 mb-4 flex items-center">
+              <img class="pr-2" src="/assets/images/icons/alert.svg" />
+              {{ 'pageTitle.alerts' | i18n }}
```
Comment (JoelDietz) [inline frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:102]: Color of the title is #111111 but should be #262124

**Review by JoelDietz — CHANGES_REQUESTED** [review]
Please add these things to add the linkage and the color

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.scss [hunk 3]
```diff
@@ -18,16 +18,41 @@
...
     gap: 40px;
   }
+
   &--overview {
     display: flex;
     gap: 24px;
+
     &-infocard {
       @apply cursor-pointer;
     }
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.scss:58]: Done

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.scss [hunk 4]
```diff
@@ -18,16 +18,41 @@
...
  ********************************************************************************/
+@import '../../../../../theme/base.scss';
+@tailwind base;
+
+.dashboard-grid {
+  @apply grid grid-cols-1 lg:grid-cols-12 gap-6;
+}
+
+.parts-section {
+  @apply col-span-12 lg:col-span-3;
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.scss:29]: Done

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html [hunk 5]
```diff
@@ -98,27 +84,56 @@ <h3 class="mb-4 flex items-center">
...
-</ng-template>
+  <ng-template #numbersMainTmp let-numbers="view" let-context="customContext">
+    <app-card-icon [label]="context.label | i18n" [icon]="context.icon" [stats]="numbers?.data"></app-card-icon>
+  </ng-template>
 
-<ng-template #numbersErrorTmp let-context="customContext">
-  <app-card-icon [label]="context.label | i18n" [icon]="context.icon" [stats]="0"></app-card-icon>
-</ng-template>
+  <ng-template #numbersErrorTmp let-context="customContext">
+    <app-card-icon [label]="context.label | i18n" [icon]="context.icon" [stats]="0"></app-card-icon>
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:134]: Done

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html [hunk 6]
```diff
@@ -98,27 +84,56 @@ <h3 class="mb-4 flex items-center">
...
       </mat-card-content>
     </mat-card>
   </section>
-</div>
 
-<ng-template #numbersMainTmp let-numbers="view" let-context="customContext">
-  <app-card-icon [label]="context.label | i18n" [icon]="context.icon" [stats]="numbers?.data"></app-card-icon>
-</ng-template>
+  <ng-template #numbersMainTmp let-numbers="view" let-context="customContext">
+    <app-card-icon [label]="context.label | i18n" [icon]="context.icon" [stats]="numbers?.data"></app-card-icon>
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:130]: Done

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html [hunk 7]
```diff
@@ -19,74 +19,60 @@
...
-      >
-      </ng-container>
-    </section>
+  <section class="parts-section">
+    <ng-container *viewContainer="
+        this.numberOfInvestigations$ | async;
+        main: numbersMainTmp;
+        error: numbersErrorTmp;
+        loading: numbersLoaderTmp;
+        customContext: { label: 'pageDashboard.totalInvestigations.label', icon: 'open_investigations_icon'}
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:51]: Done

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html [hunk 8]
```diff
@@ -19,74 +19,60 @@
...
-            label: 'pageDashboard.totalQualityAlerts.label',
-            icon: '/icons/open_investigations_icon.svg'
-          }
-        "
-      >
-      </ng-container>
-    </section>
-  </div>
-  <section class="dashboard--investigation">
+          customContext: { label: 'pageDashboard.totalAlerts.label', icon: 'alertDashBoard' }
```
[the remark this answers is not in the document]
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:62]: Done

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html (hunk 2 above)
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:102]: Done

File: frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html (hunk 1 above)
Comment (viannbreslerAccso) [reply frontend/src/app/modules/page/dashboard/presentation/dashboard.component.html:74]: Done

**Review by JoelDietz — APPROVED** [review]

**Pull request merged**